### DISCUSSION TITLE
Create Draft Pull Request from the API

### DISCUSSION THREAD

**Discussion by DannyBen** [community/community#24461] [Apps, API and Webhooks]
<p>I am creating Pull Request through the GitHub API, and was trying to submit a Pull Requets in draft mode. I followed ths <a href="https://developer.github.com/v3/pulls/#create-a-pull-request" rel="nofollow noopener">updated API Documentation for creating a pull request</a>, and took these steps:</p>
<ol>
<li>
<p>Added an Accept header to the request with </p>
<p>application/vnd.github.shadow-cat-preview+json</p>
</li>
<li>
<p>Sent this in my POST request</p>
<p>{ draft: true }</p>
</li>
</ol>
<p>But the resulting Pull Request was still a “regular” one.</p>
<p>Can anyone tell if I am doing something wrong, or perhaps this functionality is not yet available? </p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Thanks for the question.</p>
<p>According to <a href="https://developer.github.com/changes/2019-02-14-draft-pull-requests/" rel="nofollow noopener">the blog post introducing this support</a>, it appears that creating draft pull requests is limited to the v4 GraphQL API and the v3 REST API only supports draft indicators in API responses. It appears that if you want to create draft pull requests, you’ll need to use the v4 GraphQL API.</p>
<p>I hope that helps!</p>
